docs/editors/overview.md

```diff
@@ -38,181 +39,207 @@ functionality.
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Find references</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Run/Debug</td>
     <td align="center">✅</td>
     <td align="center"></td>
     <td align="center"></td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Find implementations</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Rename symbol</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Code actions</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Worksheets</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Document symbols</td>
     <td align="center">✅</td>
     <td align="center">Flat</td>
     <td align="center">Flat</td>
     <td align="center">✅</td>
+    <td align="center">Flat</td>
   </tr>
   <tr>
     <td>Workspace symbols</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Formatting</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Folding</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center"> </td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Highlight</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Metals Extensions</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">Status bar, Input box, Decoration protocol, Did focus</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
      <td>Organize imports</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">❌</td>
   </tr>
   <tr>
      <td>Implicit decorations</td>
      <td align="center">✅</td>
      <td align="center">Shown in hover</td>
      <td align="center">✅</td>
      <td align="center">✅</td>
+     <td align="center">❌</td>
```

Comment:
This will most likely change once the support for inlay hints (#5983) lands in Metals. :slightly_smiling_face:

Reply:
@keirlawson Now that #5983 with inlay hints is merged the decorations work in Helix! :tada: Big thanks to the whole Metals team, @jkciesluk for pushing it forward, @kasiaMarek and @tgodzik for reviewing this significant rework. :slightly_smiling_face:  This will also help with the other editors without dedicated client plugin, like Zed. :+1: